srcpkgs/fontconfig/template

```diff
@@ -23,6 +23,8 @@ post_install() {
 	if [ "$CROSS_BUILD" ]; then
 		sed -i 's,\(Cflags: -I${includedir}\).*,\1,g' ${DESTDIR}/usr/lib/pkgconfig/fontconfig.pc
 	fi
+	vinstall ${FILESDIR}/11-lcdfilter-default.conf 644 usr/share/fontconfig/conf.avail
+	ln -s /usr/share/fontconfig/conf.avail/11-lcdfilter-default.conf ${DESTDIR}/etc/fonts/conf.d/
```

Comment:
This is not the right way to handle this configuration snippet. In fact, fontconfig improperly installs several snippets in `/etc/fonts/conf.d`. The point of `/etc/fonts/conf.d` is to allow site-specific customization of fontconfig, but symlinks managed by the package will be recreated if the user deletes them and reinstalls or updates the package. Also, if the user removes the managed links, `xbps-pkgdb` will complain about broken symlinks when checking package integrity.

The right way to manage these symlinks is create an `INSTALL` script, run only for the initial installation and not an upgrade.